src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -477,8 +476,8 @@ TEST_F(LiveFunctionsDataViewTest, ContextMenuActionsAreInvoked) {
     }
     EXPECT_CALL(app_, GetCaptureData).WillRepeatedly(testing::ReturnRef(*capture_data_));
 
-    EXPECT_CALL(app_, GetAllTimersForHookedFunction)
-        .WillRepeatedly(testing::Return(kTimerPointers));
+    AddTimersToThreadTrackDataProvider(capture_data_->GetThreadTrackDataProvider());
+    capture_data_->OnCaptureComplete();
```

Comment:
I would prefer add the `AddTimersToThreadTrackDataProvider` method to the text fixture `LiveFunctionsDataViewTest` and change it to include `capture_data_->OnCaptureComplete()`.

Reply:
I think it would be even better to remove the function altogether and simply add the three lines that add the timers and call `OnCaptureComplete` to `GenerateTestCaptureData`. 

Thanks for making me think about this.